deploy/llms/Dockerfile.fastchat-worker-ray

```diff
@@ -0,0 +1,7 @@
+ARG BASE_IMAGE_VERSION="vllm-v0.1.0"
+FROM kubeagi/arcadia-fastchat-worker:${BASE_IMAGE_VERSION}
+
+# Overwrite the python version to pass the version check
+# for now, ray: 2.9.0, python: 3.9.x
+# this utils.py file is from ray 2.9.0 ray-ml image
+COPY utils.py /usr/local/lib/python3.9/dist-packages/ray/_private/utils.py
```

Comment:
So this `utils.py` is doing the version check,right?

Reply:
yes, in the check_version_info function